plugins/woocommerce-blocks/assets/css/abstracts/_mixins.scss

```diff
@@ -195,8 +195,6 @@ $fontSizes: (
 	overflow-wrap: anywhere;
 	// Safari supports word-break.
 	word-break: break-word;
-	// IE11 doesn't support overflow-wrap neither word-break: break-word, so we fallback to -ms-work-break: break-all.
-	-ms-word-break: break-all;
 }
 
 // Add support for content alignment classes
```

Comment:
No need to do it in this PR (it can be a follow-up), but I think this mixin isn't used anywhere. Maybe it's time to remove it?

Reply:
Yes I saw that however since this is a utility mixin, I thought perhaps it might be used later somewhere and thus I didn't remove it.

Reply:
Hmmm, I would say that if it's not used anywhere, it's better to remove it. If we ever want to use it later, it will continue to be available via git. I think that's better than keeping dead code around. What do you think?